spec/forms/hub/update_client_form_spec.rb

```diff
@@ -0,0 +1,256 @@
+require "rails_helper"
+
+RSpec.describe Hub::UpdateClientForm do
```

Comment:
if everything in here is testing an instantiated form, you could do this at the top of the spec:
```ruby
subject do
  described_class.new(params)
end
```

and then just call `.save` or `.valid?` in the tests, instead of having to also instantiate